Task10_Auth/src/main/java/reseponsesDTO/UsersDto.java

```diff
@@ -1,21 +1,45 @@
 package reseponsesDTO;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.RandomUtils;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Random;
 
 public class UsersDto {
-    private int age;
-    private String name;
-    private String zipCode;
+    private int age = RandomUtils.nextInt(1, 99);
```

Comment:
Lines 12, 13, 15: move initialization in default ctor (ctor without params)

Reply:
done